src/action/runOctoGuideAction.test.ts

```diff
@@ -1025,4 +1025,304 @@ describe("runOctoGuideAction", () => {
 			});
 		});
 	});
+
+	describe("include-associations configuration", () => {
+		describe("default behavior (excludes COLLABORATOR and OWNER)", () => {
+			it("should skip for users with OWNER author_association", async () => {
+				createMockActionInputs({
+					"include-associations":
+						"FIRST_TIMER,FIRST_TIME_CONTRIBUTOR,CONTRIBUTOR,MEMBER",
+				});
+
+				await runOctoGuideAction(
+					createMockContext(
+						createMockPayload({
+							issue: {
+								author_association: "OWNER",
+								html_url: "https://github.com/test/repo/issues/1",
+								number: 1,
+								user: { login: "owner", type: "User" },
+							},
+						}),
+					),
+				);
+
+				expect(mockCore.info).toHaveBeenCalledWith(
+					"Skipping OctoGuide rules for OWNER created issue: https://github.com/test/repo/issues/1",
+				);
+				expect(mockRunOctoGuideRules).not.toHaveBeenCalled();
+			});
+			it("should skip for users with COLLABORATOR author_association", async () => {
+				createMockActionInputs({
+					"include-associations":
+						"FIRST_TIMER,FIRST_TIME_CONTRIBUTOR,CONTRIBUTOR,MEMBER",
+				});
+
+				await runOctoGuideAction(
+					createMockContext(
+						createMockPayload({
+							issue: {
+								author_association: "COLLABORATOR",
+								html_url: "https://github.com/test/repo/issues/1",
+								number: 1,
+								user: { login: "collaborator-user", type: "User" },
+							},
+						}),
+					),
+				);
+
+				expect(mockCore.info).toHaveBeenCalledWith(
+					"Skipping OctoGuide rules for COLLABORATOR created issue: https://github.com/test/repo/issues/1",
+				);
+				expect(mockRunOctoGuideRules).not.toHaveBeenCalled();
+			});
+		});
+		describe("default associations run rules", () => {
```

Comment:
[Style] Nitty nit, most of the blocks have a blank line but this one doesn't:

```suggestion
		});

		describe("default associations run rules", () => {
```